### ISSUE TITLE
 how to delete chromebrew  

### ISSUE THREAD

**Issue by siddhantsid** [skycocker/chromebrew#3047]
i want to delete chrome-brew directory   please tell how to delete it

**Comment by vincowl** [comment]
Please have a look at #9. 
If you want to remove also installed apps, maybe not the cleanest way but here is my process `crew list installed | xargs crew remove`

**Comment by sajjaadf** [comment]
does this need to be in the wiki?

**Comment by JL2210** [comment]
@uberhacker `rm -rf /usr/local` won't remove the chroots, only the comands to enter the chroots. You can reinstall those files and your data won't be lost. The reason being, `/usr/local/chroots` is a symlink, and is not removed recursively.

**Comment by uberhacker** [comment]
I haven't installed Crouton or any other chroot in a long time.  What directory is the symlink targeting?  If that is a subdirectory of `/usr/local` also then you lose the chroot files.

**Comment by JL2210** [comment]
`/mnt/stateful_partition/crouton/chroots`

**Comment by JL2210** [comment]
@uberhacker, can you close this?

**Comment by uberhacker** [comment]
Before we close this, I need a better explanation of https://github.com/skycocker/chromebrew/issues/3047#issuecomment-451291281.  Please explain how to restore the chroots in Layman's terms.  If you can add the instructions in the wiki, that would be even better.

**Comment by JL2210** [comment]
It is in the wiki. Check the `How-to-delete-chromebrew` page.
